static/js/modules/tables.js

```diff
@@ -222,7 +222,7 @@ function compareQuery(first, second, keys) {
   return !different;
 }
 
-function pushQuery(params) {
+function pushQuery(params, push) {
```

Comment:
My only comment on this is just general advice that you can take or leave. I'd say the code would be a little more readable if it didn't have a boolean parameter but had two separate functions to call. I often consider it a potential code smell when theres a boolean parameter.

I'll leave it to you if you want to change that or not.